indico/modules/rb_new/client/js/modules/landing/Landing.jsx

```diff
@@ -34,8 +38,16 @@ class Landing extends React.Component {
     static propTypes = {
         actions: PropTypes.exact({
             setFilters: PropTypes.func.isRequired,
+            fetchUpcomingBookings: PropTypes.func.isRequired
         }).isRequired,
-        userHasFavorites: PropTypes.bool.isRequired
+        userHasFavorites: PropTypes.bool.isRequired,
+        fetchedUpcomingBookings: PropTypes.bool.isRequired,
+        hasUpcomingBookings: PropTypes.bool.isRequired,
+        upcomingBookings: PropTypes.arrayOf(PropTypes.object)
+    };
+
+    static defaultProps = {
+        upcomingBookings: null
```

Comment:
does this ever happen? i guess the default in the redux state is an empty list

Reply:
Indeed.

Reply:
`data` is `null` by default. `state = null` as default parameter in the reducer.